ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -224,6 +223,7 @@ test-suite test-storage
                        Test.Ouroboros.Storage.ImmutableDB.Primary
                        Test.Ouroboros.Storage.ImmutableDB.StateMachine
                        Test.Ouroboros.Storage.LedgerDB
+                       Test.Ouroboros.Storage.LedgerDB.DbChangelog
```

Comment:
I agree that it is useful to match locations of test and functionality. I put the Changelog tests in the location they are now after discussion with Damian and Javier. It feels to me that the DbChangelog is specific to the LedgerDB, so it might make sense to move it to LedgerDB. However, I don't have a good enough feeling yet for whether that matches the structure of the ouroboros-consensus-package.